src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -3703,6 +3787,210 @@ private void SetDefaultScaleFactor(WorkspaceViewModel defaultWorkspace)
             }
         }
 
+        private DirectoryInfo GetFallBackDocPath()
+        {
+            DirectoryInfo fallbackDocPath = null;
+            const string FALLBACK_DOC_DIRECTORY_NAME = "fallback_docs";
+            var pathManager = Model.PathManager;
+
+            if (!string.IsNullOrEmpty(pathManager.DynamoCoreDirectory))
+            {
+                var docsDir = new DirectoryInfo(Path.Combine(pathManager.DynamoCoreDirectory, System.Threading.Thread.CurrentThread.CurrentCulture.ToString(), FALLBACK_DOC_DIRECTORY_NAME));
+                if (!docsDir.Exists)
+                {
+                    docsDir = new DirectoryInfo(Path.Combine(pathManager.DynamoCoreDirectory, "en-US", FALLBACK_DOC_DIRECTORY_NAME));
+                }
+                fallbackDocPath = docsDir.Exists ? docsDir : null;
+            }
+
+            if (!string.IsNullOrEmpty(pathManager.HostApplicationDirectory))
+            {
+                //when running over any host app like Revit, FormIt, Civil3D... the path to the fallback_docs can change.
+                //e.g. for Revit the variable HostApplicationDirectory = C:\Program Files\Autodesk\Revit 2023\AddIns\DynamoForRevit\Revit
+                //Then we need to remove the last folder from the path so we can find the fallback_docs directory.
+                var hostAppDirectory = Directory.GetParent(pathManager.HostApplicationDirectory).FullName;
+                var docsDir = new DirectoryInfo(Path.Combine(hostAppDirectory, System.Threading.Thread.CurrentThread.CurrentCulture.ToString(), FALLBACK_DOC_DIRECTORY_NAME));
+                if (!docsDir.Exists)
+                {
+                    docsDir = new DirectoryInfo(Path.Combine(hostAppDirectory, "en-US", FALLBACK_DOC_DIRECTORY_NAME));
+                }
+                fallbackDocPath = docsDir.Exists ? docsDir : null;
+            }
+            return fallbackDocPath;
+        }
+        private DirectoryInfo GetNodeHelpDocPath()
+        {
+            const string SHARED_DOCS_DIRECTORY_NAME = "NodeHelpSharedDocs";
```

Comment:
There should be the same const somewhere  else, maybe worth moving to a const under DynamoConfiguration class?